harness/determined/cli/experiment.py

```diff
@@ -1294,6 +1304,23 @@ def experiment_id_arg(help: str) -> Arg:  # noqa: A002
                         Arg("max_slots", type=none_or_int, help="max slots"),
                     ],
                 ),
+                Cmd(
+                    "log-retention-days",
```

Comment:
Keeping it in line with the function name, I would call this `log-retention` rather than `log-retention-days`.
Also it's strange to me that this function has two optional arguments. Perhaps `days` should be a positional argument that is `int_or_none` and then `--forever` can be an optional argument.

Reply:
done